router/core/graphql_handler.go

```diff
@@ -186,6 +188,20 @@ func (h *GraphQLHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 			return
 		}
 
+		if errs := ctx.SubgraphErrors(); errs != nil {
+			w.Header().Set("Cache-Control", "no-cache")
```

Comment:
Please use `Cache-Control: no-store, no-cache, must-revalidate` to avoid any cache at all.